From d0ecd95e8935a9fe3e94921d10bfb153f5e0ce52 Mon Sep 17 00:00:00 2001
From: Nicolas Dechesne <nicolas.dechesne@linaro.org>
Date: Wed, 28 Aug 2019 17:54:50 +0200
Subject: [PATCH] Makefile: Use pkg-config for libxml2 detection.

xml2-config does not work. Use pkgconfig to set CPPFLAGS and LIBS.

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Nicolas Dechesne <nicolas.dechesne@linaro.org>
---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/Makefile
+++ b/Makefile
@@ -1,14 +1,14 @@
 OUT := qdl
 
-CFLAGS := -O2 -Wall -g `xml2-config --cflags`
-LDFLAGS := `xml2-config --libs` -ludev
-prefix := /usr/local
+CFLAGS += -O2 -Wall -g `pkg-config --cflags libxml-2.0`
+LDFLAGS += `pkg-config --libs libxml-2.0` -ludev
+prefix ?= /usr/local
 
 SRCS := firehose.c qdl.c sahara.c util.c patch.c program.c ufs.c
 OBJS := $(SRCS:.c=.o)
 
 $(OUT): $(OBJS)
-	$(CC) -o $@ $^ $(LDFLAGS)
+	$(CC) $(CFLAGS) -o $@ $^ $(LDFLAGS)
 
 clean:
 	rm -f $(OUT) $(OBJS)
